File: task3.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the numbers with a space: ");
        String entryNums = scanner.nextLine();
        String[] nums = entryNums.split(" ");

        for (String num : nums) {
            if (num.length() != 3) {
                System.out.println(num + " - It's not a three-digit number");
                continue;
            }

            int number = Integer.parseInt(num);
            if (isPalindrome(number)) {
                System.out.println(num + " - Is Palindrome");
            } else {
                System.out.println(num + " - Is not Palindrome");
            }
        }
    }

    public static boolean isPalindrome(int num) {
        String numStr = String.valueOf(num);
        return numStr.equals(new StringBuilder(numStr).reverse().toString());
    }
}

File: Task3.js
function isPalindrome(num) {
    let num_str = num.toString();
    return num_str === num_str.split('').reverse().join('');
}

let entry_nums = prompt("Enter the numbers with a space: ");
let Nums = entry_nums.split(" ");

for (let num of Nums) {
    if (num.length !== 3) {
        console.log(num + " - It's not a three-digit number");
        continue;
    }
    num = parseInt(num);
    if (isPalindrome(num)) {
        console.log(num + " - Is Palindrome");
    } else {
        console.log(num + " - Is not Palindrome");
    }
}
